### DISCUSSION TITLE
Nice addon for HA

### DISCUSSION THREAD

**Discussion by gauzilla** [sbidy/wiz_light#132] [General]
Besides scheduling lights to go on and off, is anyone using it for anything else?

**Comment by ChrisLizon** [comment]
I have multiple wiz bulbs that come on/off using one z-wave wall switch. If it's after sundown my desk lamp comes on. If it's daytime my overhead light comes on. They are both configured to do cercadian lighting via the [Circadian lighting Custom component](https://github.com/claytonjn/hass-circadian_lighting)

**Comment by gauzilla** [comment]
New to the HA so I am trying to learn the ropes.  We mainly bought the WiZ lights to provide different colors for the various holidays.  Need to figure out how to automate that so we dont have to manually change it.

**Comment by code-in-progress** [comment]
I currently have 58 bulbs. Every room in the house has a minimum of 5 scenes (defined in HA) based upon time of day and they only get turned on/off by either voice commands (via Alexa and/or Mycroft) or through motion/contact sensors. They adjust automatically to the time of day and set white level and brightness automatically. I use them for alerts as well (the Christmas effect is great for that). So, if I'm in my office and there's motion detected on my front porch, one of my desk lights gets set to the Christmas effect for 1 minute and then returns to its previous state.

**Comment by danielr1996** [comment]
I use a combination of home assistant, node red and a shelly 1 to turn all lights in room on/off with just one button. The shelly is in detached mode so it doesn't actually cut the power but only triggers a state change in ha, in node red I made a rule to then turn the lights on or off.

**Comment by abigdeel** [comment]
So my approach has been to use the circadian rhythm setting in the Wiz app for the rooms that have these lights. And I control on/off via the dumb wall switch sometimes.

I also have a Conbee II integrated with HA so I use some Ikea dimmers as well to dim or brighten the lights. I use them to adjust the color of the lights as well, even though it's just the two button dimmer switch. If I hold both buttons down together for ~5 seconds, then it changes the function of the buttons to adjust the color of the lights instead. My lights dim and brighten to indicate that the dimmer has switched modes. That way I can use the dimmer for brightness and also to make the lights warmer/cooler (I just have the tunable white lights). And if I want to set it back to the Wiz circadian rhythm, then I have a few clever ways to have the lights go back into that mode.

I wouldn't advise using an Ikea dimmer like this unless you're prepared to account for a few tricky behaviours of the two button dimmer switch from Ikea (if you depress both buttons and then release, only one will show as having released)
